File: src/com/leetcode/Easy/MissingNumber.java
package com.leetcode.Easy;

import java.util.Arrays;

// 268. Missing Number
public class MissingNumber {

    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        for (int i = 0; i < nums.length - 1; i++) {
            if ((nums[i] + 1) != nums[i + 1])
                return (nums[i] + 1);
        }
        return (nums[nums.length - 1] + 1);
    }
}

File: JavaScript/Easy/MissingNumber.js
/**
 * 268. Missing Number
 * @param {number[]} nums
 * @return {number}
 */
const missingNumber = function (nums) {
  nums.sort((a, b) => a - b);
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] != i)
      return i;
  }
  return nums.length;
};